API/Services/PhotoService.cs

```diff
@@ -1,18 +1,21 @@
 using System.Threading.Tasks;
 using CloudinaryDotNet;
 using CloudinaryDotNet.Actions;
+using Entities;
 using Helpers;
 using Interfaces;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 
 namespace Services
 {
   public class PhotoService : IPhotoService
   {
     private readonly Cloudinary _cloudinary;
+    private readonly IUserRepository _userRepository;
 
-    public PhotoService(IOptions<CloudinarySettings> config)
+    public PhotoService(IOptions<CloudinarySettings> config, IUserRepository userRepository)
     {
         var acc = new Account
```

Comment:
Not sure the logic here, just use a default Account without any values populated?